service.tsx

```diff
@@ -0,0 +1,12 @@
+const service = {
+    getId: function ({ item }) {
+        const regex = /https:\/\/swapi.dev\/api\/vehicles\/\d*/;
+        const uniform_url = item.url.match(regex)[0];
+        console.log(uniform_url);
+        return parseInt(
+            uniform_url.split("/")[uniform_url.split("/").length - 1]
+        );
```

Comment:
(nie mam teraz jak tego przetestowac, więc pewnie nie zadziala, ale...)
```suggestion
        const regex = /https:\/\/swapi.dev\/api\/vehicles\/(\d+)/;
        const id = url.match(regex)[1];
        return id ? parseInt(id) : undefined;
```